packages/common/src/services/extension.service.ts

```diff
@@ -162,6 +166,21 @@ export class ExtensionService {
         this.translateItems(this.sharedService.allColumns, 'nameKey', 'name');
       }
 
+      // Row Based Edit Plugin
+      if (this.gridOptions.enableRowBasedEdit) {
+        this._rowBasedEdit = this._rowBasedEdit || new SlickRowBasedEdit(this.pubSubService, this.gridOptions.rowBasedEditOptions);
+        const gridService = this.lazyGridService?.();
+        if (!gridService) {
+          throw new Error('[Slickgrid-Universal] requires a GridService to be configured and available');
```

Comment:
I think we should add RowBaseEdit somewhere in the error message to advise the throw caller to the user